extensions/undertow/runtime/src/main/java/io/quarkus/undertow/runtime/KnownPathResourceManager.java

```diff
@@ -144,10 +148,12 @@ public List<Resource> list() {
                             try {
                                 Resource resource = underlying.getResource(i);
                                 if (resource == null) {
-                                    throw new RuntimeException("Unable to get listed resource " + i + " from directory " + path
-                                            + " for path " + slashPath + " from underlying manager " + underlying);
+                                    log.warnv(
```

Comment:
Shouldn't this be `errorv` instead?

Reply:
BTW I think logging the underlying manager is irrelevant here

Reply:
Done